### ISSUE TITLE
NTR: Trabecular bladder

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4665]
**Preferred term label:**
Trabecular bladder

**Synonyms**
Bladder trabeculation

**Definition (free text, please give PubMed ID)**
Bladder trabeculation is caused by morphological and histological changes due to hypertrophy and hyperplasia of the bladder muscle and the infiltration of the connective tissue. 
PMID:28904941

**Parent term (use hpo.jax.org/app)**
HP_0025487 'Abnormality of bladder morphology'

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.omim.org/entry/614080?search=614080&highlight=614080

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

**Comment by pnrobinson** [comment]
Trabeculae are muscular projections that protrude into the lumen of the bladder, criss-crossing the walls of the bladder on its inner surface.
